mmv1/templates/terraform/examples/apigee_endpoint_attachment_basic.tf.erb

```diff
@@ -0,0 +1,33 @@
+data "google_client_config" "current" {}
+
+resource "google_compute_network" "apigee_network" {
+  name = "apigee-network"
```

Comment:
Each resource name should be wrapped in a ruby tag, see https://github.com/GoogleCloudPlatform/magic-modules/blob/master/mmv1/templates/terraform/examples/pubsub_lite_subscription_basic.tf.erb